demos/collection/multitable.php

```diff
@@ -38,23 +38,24 @@ public function setModel(\atk4\data\Model $model, $route = [])
 
         while ($selections && $id = array_shift($selections)) {
             $path[] = $id;
-            $model->tryLoad($id);
-            if (!$model->loaded()) {
+            $pushModel = $model->newInstance();
```

Comment:
```suggestion
            $pushModel = clone $model;
```

otherwise we will loose scope if set (or other settings not set in `Model::init()`)

@georgehristov should we remove `newInstance()` from Model or change it, so clone is returned instead?

Reply:
clone $model != $model->newInstance

Using newInstance is not supposed to take over the scope but only use the same connection.

Reply:
ok - it is not clone, as it does not clone ID etc.

so `$obj->newInstance()` is strictly equivalent to `new get_class($obj)()` (only theoretical, new expr is not supported)?

if model is dynamic like now, ie. defined with scopes, what is the advantage of this function?

Reply:
As I mentioned - also the same connection is used. Actual code speaks for itself:

```
public function newInstance(string $class = null, array $options = [])
    {
        $model = (self::class)::fromSeed([$class ?? static::class], $options);

        if ($this->persistence) {
            return $this->persistence->add($model);
        }

        return $model;
    }
```

Reply:
@mvorisek - clone $model is not fully working here too. It prevents from going deeper if a folder is click in right column. You should be able to click on the rightmost columns until you see no records. Was working using $model->newInstance()

Reply:
this is how it should look
<img width="1035" alt="Screen Shot 2020-10-29 at 12 11 32 PM" src="https://user-images.githubusercontent.com/2204478/97600910-f09aa480-19df-11eb-83b4-234059901e12.png">

Reply:
@ibelar I tested yesterday, I almost wanted to submit an issue... But it works! Ok - see https://github.com/atk4/ui/pull/1523 , we simply revert my change.